client/src/FileUpload.js

```diff
@@ -1,14 +1,38 @@
-const FileUpload = () => {
-    return (
-        <div className="ui container">
-            <div className="content">
-                <h3 className="ui green header">File Upload</h3>
-            </div>
-            <div className="content">
-                <input type="file" name="filename"/>
-            </div>
-        </div>
-    );
-};
+import React from 'react';
+
+class FileUpload extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            fileOpened: false
+        };
+        
+        this.readIn = this.readIn.bind(this);
+    }
 
-export default FileUpload;
\ No newline at end of file
+    readIn(event) {
+        console.log(event.target.files[0]);
+        this.setState({ fileOpened: true });
```

Comment:
is this used `fileOpened` state used anywhere? i'm not immediately seeing it mentioned anywhere except for setting it to true or false.

Reply:
Sorry I forgot to reply here last week. To follow up with what I said via Slack, I originally thought of changing some styling based on whether or not a file had already been uploaded. I think there still might be a use case for this, so I left the code in since it might come in handy somewhere down the line as we wrap up the project. If it ends up not being used, it can certainly be scrapped entirely. However, I think for now it's worth keeping in.